Tools/scripts/run_coverage.py

```diff
@@ -0,0 +1,295 @@
+#!/usr/bin/env python3
+
+"""
+Runs tests with gcov coverage support.
+
+ AP_FLAKE8_CLEAN
+"""
+import argparse
+import os
+import time
+import shutil
+import subprocess
+import sys
+
+os.environ['PYTHONUNBUFFERED'] = '1'
+os.set_blocking(sys.stdout.fileno(), True)
```

Comment:
I wonder if we did that after every job whether we might have better luck.

Also, this might need to be done on stderr too.